### ISSUE TITLE
latex builder fails in case no caption option is provided to toctree directive

### ISSUE THREAD

**Issue by splendido** [sphinx-doc/sphinx#2465]
`sphinx@1.4.1`
`docutils@0.12`

[this line](https://github.com/sphinx-doc/sphinx/blob/master/sphinx/builders/latex.py#L140) is trying to check whether a caption option was provided to the `toctree` directive or not before trying to get the option value.
But at the moment if fails when no caption is provided making `docutils` throw the following error:

``` shell
  File "/usr/local/lib/python2.7/dist-packages/sphinx/cmdline.py", line 244, in main
    app.build(opts.force_all, filenames)
  File "/usr/local/lib/python2.7/dist-packages/sphinx/application.py", line 287, in build
    self.builder.build_update()
  File "/usr/local/lib/python2.7/dist-packages/sphinx/builders/__init__.py", line 246, in build_update
    self.build(['__all__'], to_build)
  File "/usr/local/lib/python2.7/dist-packages/sphinx/builders/__init__.py", line 322, in build
    self.write(docnames, list(updated_docnames), method)
  File "/usr/local/lib/python2.7/dist-packages/sphinx/builders/latex.py", line 130, in write
    doctree.settings.contentsname = self.get_contentsname(docname)
  File "/usr/local/lib/python2.7/dist-packages/sphinx/builders/latex.py", line 140, in get_contentsname
    if toctree['caption']:
  File "/usr/local/lib/python2.7/dist-packages/docutils/nodes.py", line 567, in __getitem__
    return self.attributes[key]
KeyError: 'caption'
```

The problem can be easily fixed by substituting the very same line with:

``` python
if 'caption' in toctree:
```

Please let me know if you'd like me to prepare a PR with the proposed fix.

**Comment by tk0miya** [comment]
I'll take a look.

**Comment by tk0miya** [comment]
Fixed at 88322af
Thank you for report!

**Comment by splendido** [comment]
:+1:
